index.html

```diff
@@ -1,60 +1,87 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 	<meta charset="utf-8">
-	<title>intuit form</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<title>HTML FORM</title>
 </head>
 <body>
-<div class="wrapper">
-    <section>
-	  <button class="btn"><i class="fa fa-lock"> Sign In</i></button>
-	  <p>Already have an intuit account</p>
-	  <a href="#">I forgot my User ID or Password</a>
+	<div class="wrapper">
+		<!-- Top Menu -->
+		<section class="menu-right-top">
+			<div class="pull-to-left">
+				<p>Already have an intuit account?</p>
+				<a href="#">I forgot my User ID or Password</a>
+			</div>
+			<div class="pull-to-right">
+				<button class="btn"><i class="fa fa-lock"> Sign In</i></button>
+			</div>
+		</section>
+
+		<div class="give-me-space"></div>
+
+		<div class="content">
+			<!-- Logos Section -->
+			<section class="logo-section">
+				<div class="main-logo">
+					<img src="img/logo-intuit.png" alt="intuit logo">
+				</div>
+				<div class="other-logos">
+					<img src="img/logo-mint.png" alt="mint logo"><span class="logo-name"> mint</span>
+					<img src="img/logo-quickbooks.png" alt="quickbooks logo"><span class="logo-name"> quickbooks</span>
+					<img src="img/logo-turbotax.jpg" alt="turbotax logo"><span class="logo-name"> turbotax</span>
```

Comment:
`SUGGESTION`

Consider changing the logo to look more like the original site.

| Current | Preferred |
| :-----: | :-----: |
| ![turbotax logo](https://github.com/sumancrest0001/HTMLforms/raw/master/img/logo-turbotax.jpg) | ![turbotax logo](https://accounts.intuit.com/IUS-Plugins/v2/images/tt-favicon@2x.svg) |